content/docs/2.11/operate/prometheus.md

```diff
@@ -16,6 +16,7 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 - `keda_scaler_errors` - The number of errors that have occurred for each scaler.
 - `keda_scaler_errors_total` - The total number of errors encountered for all scalers.
 - `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObejct.
+- `keda_scaled_object_paused` - The number of paused scaledObject (using the annotation detailed [here](https://keda.sh/docs/latest/concepts/scaling-deployments/#pause-autoscaling))
```

Comment:
it is not number of paused scaledobjects, but it marks whether the SO is paused or not.

Move this to line 14, so it is next to activity

Reply:
Alright I will do that.

Reply:
@geoffrey1330 please fix this, so we can go ahead and merge the feature